Feature lint windows script

leszekhanusz opened the pull request:
`style.cmd` file to check the style on Windows.

Adding a `CONTRIBUTING.md` file with documentation on how to run the style and how to add a pre-commit hook to verify the style automatically at each commit.

leszekhanusz commented on nataili/inference/compvis.py:
```diff
@@ -263,11 +260,11 @@ def sample(init_data, x, conditioning, unconditional_conditioning, sampler_name)
             prompt_tokens = re.findall("<([a-zA-Z0-9-]+)>", prompt)
             if prompt_tokens:
-                self.process_prompt_tokens(prompt_tokens, model)
+                self.process_prompt_tokens(prompt_tokens, self.model)
```
Note: Here instead of a refactor I changed `model` to `self.model`, I assume this was a bug as `model` was not defined.

leszekhanusz commented:
@hlky @db0 
Once this PR has been merged, please follow the instructions in the `CONTRIBUTING.md` file to add a pre-commit hook which will verify the style for each of your commits.
It is especially important if you push directly to the main branch.

hlky commented:
Do not make changes that are unrelated to your pull request.

`Note: Here instead of a refactor I changed model to self.model, I assume this was a bug as model was not defined.`

You assumed wrong, this does not fix the issue. I have fixed the issue.

db0 commented:
@leszekhanusz thanks for the style fix. Could you please resubmit just the style and I'll merge. If possible please look into making a `style.py` or `style.cmd` so we can run it ourselves as well and you don't need to keep repairing it. Thanks

hlky commented:
@leszekhanusz Can you resubmit this with only the `CONTRIBUTING.md` and `style.*` changes please.

leszekhanusz commented:
> Do not make changes that are unrelated to your pull request.

You're right, that would have been better

> `Note: Here instead of a refactor I changed model to self.model, I assume this was a bug as model was not defined.`
> 
> You assumed wrong, 

I assumed right since I just said that I assumed there was a bug, I did NOT say that my change will fix this. This comment here was there so that you could look into it, I did not look further.

> this does not fix the issue.

I never said it would fix the issue

> I have fixed the issue.

Great

leszekhanusz commented:
> @leszekhanusz Can you resubmit this with only the `CONTRIBUTING.md` and `style.*` changes please.

Alright
